services/web/client/source/class/osparc/auth/ui/LoginView.js

```diff
@@ -84,7 +84,6 @@ qx.Class.define("osparc.auth.ui.LoginView", {
 
       const loginBtn = this.__loginBtn = new osparc.ui.form.FetchButton(this.tr("Sign in"));
       loginBtn.addListener("execute", () => {
-        loginBtn.setFetching(true);
         this.__login(loginBtn);
```

Comment:
__login method has no need to get the button as argument

Reply:
done